src/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(-n + 4) {
```

Comment:
There is an issue with the selector ".stars--4 :nth-child(-n + 4)". The space between ".stars--4" and ":nth-child(-n + 4)" indicates that ":nth-child(-n + 4)" is targeting children of any element inside ".stars--4", not direct children. If you intend to target direct children, remove the space.